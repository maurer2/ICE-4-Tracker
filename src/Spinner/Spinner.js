import React from 'react';

import { ReactComponent as SpinnerSVG } from './Spinner.svg';
import styles from './Spinner.module.css';

const Spinner = () => {
  return (
    <SpinnerSVG className={ styles.spinner } />
  )
}

export default Spinner;
