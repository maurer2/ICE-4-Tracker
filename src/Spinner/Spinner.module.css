.spinner {
  display: block;
  height: 100%;
}
