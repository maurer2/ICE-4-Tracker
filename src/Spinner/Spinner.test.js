import React from 'react';
import Spinner from './Spinner';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

Enzyme.configure({ adapter: new Adapter() });

describe('Spinner', () => {
  const wrapper = shallow(
    <Spinner/>
  );

  it('Spinner should match snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
