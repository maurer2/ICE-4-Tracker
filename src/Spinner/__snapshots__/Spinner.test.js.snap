// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Spinner should match snapshot 1`] = `<ForwardRef />`;
